core/src/main/javascript/app/components/SlidePanel.js

```diff
@@ -7,7 +7,8 @@ import classNames from "classnames";
 type Props = {
   classes: any,
   title: string,
-  children: any
+  children: any,
+  open: boolean
```

Comment:
I am not sure why you extracted the "open" parameter to the props, but I have no objection.